src/common/hook/useQueue/utils/publish.tsx

```diff
@@ -136,10 +125,10 @@ export const groupDocumentsIntoJobs = async (
   const verifiableDocuments = rawDocuments.filter((doc) => doc.type === "VERIFIABLE_DOCUMENT");
   const groupedVerifiableDocuments = groupBy(verifiableDocuments, "contractAddress");
   const verifiableDocumentsWithDocumentStore = { ...groupedVerifiableDocuments };
-  delete verifiableDocumentsWithDocumentStore[identifyProofType.DnsDid];
+  delete verifiableDocumentsWithDocumentStore[IdentityProofType.DNSDid];
   const verifiableDocumentsWithDnsDid =
-    Object.keys(groupedVerifiableDocuments).indexOf(identifyProofType.DnsDid) >= 0
-      ? [...groupedVerifiableDocuments[identifyProofType.DnsDid]]
+    Object.keys(groupedVerifiableDocuments).indexOf(IdentityProofType.DNSDid) >= 0
+      ? [...groupedVerifiableDocuments[IdentityProofType.DNSDid]]
       : [];
```

Comment:
if i read the code correctly, it wont come into this array cos it didn't cater to the type in the getIssuerAddress().

Reply:
didn't quite understand

Reply:
ok let me look into this more